docs/pages/misc-pages/grpc-v1.md

```diff
@@ -49,6 +49,8 @@ interface as the grpc v1 node client:
 
 ## GRPCv1-Client
 
+_Only accessible prior to version 10_
```

Comment:
Maybe this should be phrased differently, since it is referring to version 10 of nodejs sdk, which is not going to be a thing.